### PR TITLE
Users will see all management errors at once

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 
## Issue
https://github.com/Fliplet/fliplet-studio/issues/5439#issuecomment-575641157

## Description
Users will see all management errors at once

## Backward compatibility

This change is fully backward compatible.

### CODE REVIEW

**Comment by squallstar** [comment]
@YaroslavOvdii see linked warning from Codacy https://app.codacy.com/manual/Fliplet/fliplet-widget-dynamic-lists/pullRequest?prid=4866220

**Comment by YaroslavOvdii** [comment]
@squallstar 
I've looked at the warning it says that this issue was fixed.
And there is no such line left in code.
As far as I understand there is no error left, or should I return that part of the code?

**Comment by squallstar** [comment]
@YaroslavOvdii do you see this on Codacy?

image

If the line does not exist anymore then you can ignore the warning, I'm not sure why it didn't update there.

**Comment by YaroslavOvdii** [comment]
@squallstar 

> @YaroslavOvdii do you see this on Codacy?

Yes, I can see this part on Codacy. 

> If the line does not exist anymore then you can ignore the warning, I'm not sure why it didn't update there.

Yes, I removed this line from the code because I remade the error handler for the Entry Management tab.

**Comment by squallstar** [comment]
@YaroslavOvdii thanks for confirming, you can ignore the warning then.

File: js/interface-lists.js [hunk 1]
```diff
@@ -381,82 +350,70 @@ function attahObservers() {
...
+        if (widgetData.addEntry) {
+          selectedPermissions.push('addPermissions');
+        }
 
-            if (!linkAddEntryProvider || !linkEditEntryProvider) {
-              withError = true;
-              linkProviderInit();
-            }
+        if (widgetData.editEntry) {
+          selectedPermissions.push('editPermissions');}
```
Comment (tonytlwu) [inline js/interface-lists.js]:
```suggestion
          selectedPermissions.push('editPermissions');
        }
```

Comment (tonytlwu) [reply js/interface-lists.js]: @sofiiakvasnevska We just spoke about this on Friday. The developer team should carefully review for things like this.

**Pull request merged**